src/pages/en/about/security-notice.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Security Notice
+description: If you find a vulnerability in one of our project, please report it using the procedure detailed here
+layout: /src/layouts/Page.astro
+---
+
+This is the security notice for all QuiltMC repositories. The notice explains how vulnerabilities should be reported. We also support the [security.txt](/.well-known/security.txt) standard.
```

Comment:
I've added "security" in front of vulnerabilities here and in some other places to make it clearer for people who might not be familiar with the terminology.
```suggestion
This is the security notice for all QuiltMC repositories. The notice explains how security vulnerabilities should be reported. We also support the [security.txt](/.well-known/security.txt) standard.
```